Source/UTags/Public/Tags.h

```diff
@@ -6,22 +6,19 @@
 #include "EngineUtils.h"
 #include "Tags.generated.h"
 
-/*
-* FTagData - the structure of the tag data
-* TagType;Key1,Value1;Key2,Value2;[..]
-*/
+
```

Comment:
/*
* FTagData - the structure of the tag data
* TagType;Key1,Value1;Key2,Value2;[..]
*/